tests/features/splash_screen.feature

```diff
@@ -25,19 +24,19 @@ Feature: Splash Screen features
     And I should see the link "Français"
     And I should see the link "Български"
 
-  @api
-  Scenario: Links on splash screen pages are correct
+  # Regression test for a bug that broke the Portuguese (pt-pt) link.
+  # See https://webgate.ec.europa.eu/CITnet/jira/browse/NEXTEUROPA-5585
+  Scenario: Test language with a custom prefix
     Given I am logged in as a user with the 'administrator' role
-    When I go to "admin/config/regional/language/edit/fr"
-    And I fill in "edit-prefix" with "fr-prefix"
+    And the "pt-pt" language is available
```

Comment:
Why we don't use the existing function  {{these following languages are available}} ?